corehq/apps/cloudcare/templates/cloudcare/partials/menu/bootstrap5/breadcrumbs.html

```diff
@@ -6,8 +6,10 @@
 
 <script type="text/template" id="breadcrumb-list-template">
   <nav class="breadcrumb-nav" aria-label="Breadcrumb">
-    <ol class="breadcrumb">  {# todo B5: css:breadcrumb #}
-      <li class="js-home breadcrumb-text" role="link" tabindex="0">{% trans 'Home' %}</li>
+    <ol class="breadcrumb">
+      <li class="js-home breadcrumb-item" role="link" tabindex="0">
+        {% trans 'Home' %}
+      </li>
```

Comment:
Why an `<a>` anchor tag for the other `breadcrumb-item`s but not this one?

Reply:
hmm, I don't remember. Docs say either with or without the anchor is fine for breadcrumbs. I probably just made the changes on two different days. I'll pull the anchor tags out, no point having extra markup that isn't needed.

Reply:
d9b82cb6f2d

Reply:
d9b82cb6f2d